pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -267,6 +260,14 @@ def init_ddp_connection(self, global_rank: Optional[int], world_size: Optional[i
             log.info(f"initializing ddp: GLOBAL_RANK: {global_rank}, MEMBER: {global_rank + 1}/{world_size}")
             torch_distrib.init_process_group(self.torch_distributed_backend, rank=global_rank, world_size=world_size)
 
+            # on rank=0 let everyone know training is starting
+            rank_zero_info(
+                f"{'-' * 100}\n"
+                f"distributed_backend={self.torch_distributed_backend}\n"
+                f"All DDP processes registered. Starting ddp with {self.world_size} processes\n"
+                f"{'-' * 100}\n"
```

Comment:
```suggestion
                f"{'-' * 100}{os.linesep}"
                f"distributed_backend={self.torch_distributed_backend}{os.linesep}"
                f"All DDP processes registered. Starting ddp with {self.world_size} processes{os.linesep}"
                f"{'-' * 100}{os.linesep}"
```

Reply:
what does this do differently? it seems unnecessary.

Reply:
Sometimes the /n doesn't work on Windows

Reply:
From
https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files

> In text mode, the default when reading is to convert platform-specific line endings (\n on Unix, \r\n on Windows) to just \n. When writing in text mode, the default is to convert occurrences of \n back to platform-specific line endings.

So we should not use `os.linesep` and let Python handle it.